[DX-955] Add use-case based SDK references across Pub/Sub pages (#3340)
ably/docs: src/pages/docs/basics/index.mdx

The [presence](/docs/presence-occupancy) feature enables clients to be aware of other clients that are currently "present" on a channel. Subscribers receive three types of updates from presence members. These are when a client joins the presence set, when they leave the presence set, and when they update an optional payload associated with each member. The payload can be used to describe their status, or attributes associated with them, such as setting their status to 'out for lunch'.

> Presence powers the [Spaces SDK](/docs/spaces/avatar) for avatar stacks and the [Chat SDK](/docs/chat/rooms/presence) for chat room member notifications. Use presence directly for custom use cases. See [product guidance](/docs/platform/products) for help choosing the right product.

Reviewer: Not sure about this one.

Author: This is the goal of the PR to avoid ambiguity of right product selection, so keeping it as is

> Presence is the underlying primitive used by the [Spaces SDK](/docs/spaces/avatar) for avatar stacks and by the [Chat SDK](/docs/chat/rooms/presence) for chat room member notifications. It can also be used directly for custom use cases. See [product guidance](/docs/platform/products) for help choosing the right product.

Reviewer 2: First sentence is 31 words and the second is passive (`It can also be used`). Style guide wants ≤25 words and active voice:

```suggestion
Presence powers the [Spaces SDK](/docs/spaces/avatar) for avatar stacks and the [Chat SDK](/docs/chat/rooms/presence) for chat room member notifications. Use presence directly for custom use cases. See [product guidance](/docs/platform/products) for help choosing the right product.
```

Author: Fixed https://github.com/ably/docs/pull/3340/commits/1a27dd4c14df6ad2f23a2ba07baaeb1b99a79004
